<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>368</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>518</width>
    <height>368</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(39, 44, 54);
color:white;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>350</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>500</width>
       <height>350</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>FreeSerif</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(39, 44, 54);
color:white;</string>
     </property>
     <property name="title">
      <string>PARAMETRES DE TRANSLATION</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignHCenter</set>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>478</width>
           <height>207</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <layout class="QFormLayout" name="formLayout">
            <property name="labelAlignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="formAlignment">
             <set>Qt::AlignCenter</set>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="styleSheet">
               <string notr="true">border-style: outset;
border-width: 2px;
border-radius: 15px;
border-color: black;
padding: 4px;
font-size:20px;
background-color: red;
color:white ;</string>
              </property>
              <property name="text">
               <string>ALPHA :</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEdit">
              <property name="styleSheet">
               <string notr="true">border-style: outset;
border-width: 2px;
border-radius: 15px;
border-color: black;
padding: 4px;
font-size:20px;
background-color: white;
color:black ;</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="styleSheet">
               <string notr="true">border-style: outset;
border-width: 2px;
border-radius: 15px;
border-color: black;
padding: 4px;
font-size:20px;
background-color: red;
color:white ;</string>
              </property>
              <property name="text">
               <string>BETA :</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="lineEdit_2">
              <property name="styleSheet">
               <string notr="true">border-style: outset;
border-width: 2px;
border-radius: 15px;
border-color: black;
padding: 4px;
font-size:20px;
background-color: white;
color:black ;</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButton">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">.QPushButton
{
border-style: outset;
border-width: 2px;
border-radius: 5px;
border-color: black;
padding: 4px;
font-size:20px;
background-color: red;
color:white 
}
QPushButton:pressed{ background-color: blue; }</string>
          </property>
          <property name="text">
           <string>VALIDER</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButton_2">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">.QPushButton
{
border-style: outset;
border-width: 2px;
border-radius: 5px;
border-color: black;
padding: 4px;
font-size:20px;
background-color: red;
color:white 
}
QPushButton:pressed{ background-color: blue; }</string>
          </property>
          <property name="text">
           <string>ANULER</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
